Dockerfile

```diff
@@ -1,5 +1,9 @@
-# Use latest jboss/base-jdk:11 image as the base
-FROM jboss/base-jdk:11
+FROM eclipse-temurin:17.0.4.1_1-jdk-centos7
```

Comment:
If we do that switch, we will have to figure out how to name our images as I'd like to be able to provide JDK11 and 17 versions of the images.

We will likely have 1 a single Dockerfile and have the base image (either `eclipse-temurin:17-centos7` or `eclipse-temurin:11-centos7`) set as a parameter when the GitHub action builds the image.

The names of the WildFLy images would have to reflect this. With something like

`quay.io/wildfly/wildfly:26.1.2-jdk11`
`quay.io/wildfly/wildfly:26.1.2-jdk17`
`quay.io/wildfly/wildfly:latest` would point to the "oldest" supported JDK so `26.1.2-jdk11`.

wdyt?

Reply:
I can add this in the beginning:
```
ARG jdk=11
FROM eclipse-temurin:${jdk}-centos7
```

This way 
`docker build .` gives openjdk 11.0.16.1
`docker build . --build-arg jdk=11` gives openjdk 11.0.16.1
`docker build . --build-arg jdk=17` gives openjdk 17.0.4.1
Would it be enough to set up the GitHub action?

Reply:
yes, that'd enough. thanks